src/json.rs

```diff
@@ -30,9 +32,8 @@ pub enum JsonError {
 
 #[derive(Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub enum JsonOutOfRangeError {
-    U8,
-    U64,
-    U128,
+    OutOfRangeU8,
```

Comment:
Our github actions doesn't complain for current PR state. And it's related to our current toolchain.
All lint is a success.

Reply:
We run nightly clippy [as a scheduled job](https://github.com/aurora-is-near/aurora-engine/actions/workflows/scheduled_lints.yml), not part of normal CI. We do not run as part of normal CI to not block PRs spuriously, but when we already know something will fail the scheduled job then we should try to prevent it.

Please remove the `OutOfRange` prefix.

Reply:
make check-clippy without errors.
So pls provide me cargo command to check it correctly.

Reply:
`cargo +nightly clippy --no-default-features --features=mainnet -- -D warnings` (you may need to update your nightly install to the latest one first via `rustup`)

Reply:
So I'm try reproduce
1. It will use current engine nightly: cargo clippy --no-default-features --features=mainnet -- -D warnings
**SUCCESS**
2. cargo clippy --no-default-features --features=mainnet -- -D warnings
3. cargo +nightly clippy --no-default-features --features=mainnet -- -D warnings
**FAILED**:
```
    Checking evm v0.28.0 (https://github.com/aurora-is-near/sputnikvm.git?rev=09d4fe09dcb5fcabed8c1076699c8a2e70f14c23#09d4fe09)
error: internal compiler error: unexpected concrete region in borrowck: ReStatic

```

Reply:
Very strange. I do not see this error. Try doing `rustup update nightly` before running `clippy`.

Reply:
I made clean. And run again.
```
$ make clean
$ rustc +nightly -V
rustc 1.56.0-nightly (a6ece5615 2021-08-03)
$ cargo +nightly clippy --no-default-features --features=mainnet -- -D warnings
```
Same result - SUCCESS.

Reply:
Weird. I thought it would have triggered [clippy::enum_variant_names](https://rust-lang.github.io/rust-clippy/v0.0.212/index.html#enum_variant_names) because each enum variant starts with the same `OutOfRange` prefix. But if it does not trigger then I guess it's fine.